go.mod

```diff
@@ -35,8 +35,10 @@ require (
 	kubevirt.io/qe-tools v0.1.7
 	kubevirt.io/ssp-operator/api v0.0.0
 	sigs.k8s.io/controller-runtime v0.12.2
+	k8s.io/kube-aggregator v0.23.0
```

Comment:
Can you use `v0.24.2`, which is used by other `k8s.io` dependencies?
Also please run `go mod vendor` and `go mod tidy` to sort this list alphabetically.

Reply:
sure